docs/release.md

```diff
@@ -0,0 +1,22 @@
+# Status AddOn Relesea process 
```

Comment:
It's not clear if this procedure is for creating a new release or updating exiting one with changes

Reply:
No, it's about creating a new release. The assumption is that each release is immutable moving forward.